src/chigame/games/models.py

```diff
@@ -353,3 +353,11 @@ class Message(models.Model):
 
     def __str__(self):  # may be changed later
         return "Message from " + str(self.sender) + ": " + self.content
+
+
+class Review(models.Model):
+    "Represents a game review"
```

Comment:
I have a few questions about how we want to handle reviews that could effect the way the fields of this model:
1. Can a user leave multiple reviews on a single game? This could mean using a through table or imposing uniqueness constraints.
3. Can a user update their rating or review?
4. If a user leaves a rating, do they have to write a review? If they write a review, do they have to leave a rating? Do they need to do one or the other? This could mean adding validation or adjusting the `null` arguments.
5. Are there visibility settings on reviews? This could mean adding an additional field.

Reply:
1. Yeah a user can leave multiple reviews per game. I tried making the game and user attributes ManytoMany fields but then I realized that would mean multiple games and users can have the same review. Since each review will have a primary key and we'll use foreign keys from the user and game model, that will allow us to have a user make multiple reviews per game since each review will have its own primary key. Let me know if I've got that right.
2. Yeah we can allow a user to update a review. We can reflect this in the views with a new view for specifically editing a review.
3. I think a user can make a review, rate a game, or do both. I made the review and rating attributes nullable for this. Is that correct?
4. I agree there should be visibility settings. I added an attribute to determine whether the review will be public or not. We can implement this in the game detail page in issue #261.

Reply:
1. Sounds good. I think the way you have it set up makes the most sense. 
2. Also makes sense, no issue here.
3. I think what you have works but we should add an extra validation check that ensures at least the rating or the review is not null. This can be done by adding a `clean` method to the model.
4. Thanks for adding this. 

Thought of a couple more ideas for fields to potentially add:
5. Will reviews have an optional title? I'm fine with not having one but just wanted to raise it as a possibility. 
6. I think we should have a `created_at` field, which could impact the order in which reviews are displayed. One way to do this is with `created_at = models.DateTimeField(auto_now_add=True)`

Reply:
Great, thanks for addressing these!